### DISCUSSION TITLE
two-way sync issue

### DISCUSSION THREAD

**Discussion by Sybion1411** [Dvlv/BoxBuddyRS#116] [Q&A] [answered]
Hello,

I'm a Steam Deck user, so I'm linux noob and don't want to mess up with system files. BoxBuddy is great for newbies like me, but I think there is an issue I cannot solve. I installed rclone inside Arch container, functions like sync and copy (one-way sync from Google Drive to specified folder) work perfectly, but function mount (two-way sync) doesn't work as expected - I can see that rclone connected to the cloud (it can list files inside the terminal), but it doesn't show files in file manager. Is it possible that BoxBuddy (installed from flatpak) needs more accesses than only "All user files" to show the mounted files?

**Comment by Dvlv** [comment, marked as answer]
Hello,
BoxBuddy is just a UI, issues with created boxes need to be reported to Distrobox.

>  Is it possible that BoxBuddy (installed from flatpak) needs more accesses than only "All user files" to show the mounted files?

No, since the created containers do not run inside the Flatpak sandbox.
